pyClient/zeth/joinsplit.py

```diff
@@ -666,20 +666,21 @@ def _encrypt_note(out_note: ZethNote, pub_key: EncryptionPublicKey) -> bytes:
 
 
 def receive_notes(
-        addrs_and_ciphertexts: List[Tuple[int, bytes]],
+        addrs_and_ciphertexts: List[Tuple[int, bytes, bytes]],
         sender_k_pk: EncryptionPublicKey,
-        receiver_k_sk: EncryptionSecretKey) -> Iterator[Tuple[int, ZethNote]]:
+        receiver_k_sk: EncryptionSecretKey
+) -> Iterator[Tuple[int, bytes, ZethNote]]:
     """
     Given the receivers secret key, and the event data from a transaction
     (encrypted notes), decrypt any that are intended for the receiver. Return
     tuples `(<address-in-merkle-tree>, ZethNote)`. Callers should record the
     address-in-merkle-tree along with ZethNote information, for convenience
     when spending the notes.
     """
-    for address, ciphertext in addrs_and_ciphertexts:
+    for address, commit, ciphertext in addrs_and_ciphertexts:
```

Comment:
Let's rename the `addrs_and_ciphertexts` to account for the fact that we now also get the commitment. Maybe something like `addrs_commits_ciphertexts`, or more generally `event _data`? Now, `addrs_and_ciphertexts` suggests that there are only addresses and ciphertexts tracked by this variable which is not the case anymore

Reply:
Renamed to `event_data`.  Thansk.

Reply:
Thanks @dtebbs